.vscode/cspell.json

```diff
@@ -21,11 +21,49 @@
     "*.sha512",
     "*.ico",
     "*.pri",
+    "*.trx",
+    "*.binlog",
+    "*.log",
+    "*.sln",
+    "*.user",
     ".vscode/cspell.json"
   ],
   // cspell is not case sensitive
   // Sort words alphabetically to make this list easier to use
-  "words": ["azsdk", "pwsh"],
+  "words": [
+    "apos",
+    "aragorn",
+    "azsdk",
+    "blazor",
+    "contoso",
+    "deserializes",
+    "diagnoser",
+    "epsg",
+    "expando",
+    "finalizer",
+    "finalizers",
+    "ints",
+    "lucene",
+    "mgmt",
+    "nunit",
+    "odata",
+    "otel",
+    "overridden",
+    "parallelization",
+    "pkcs",
+    "pwsh",
+    "referer",
+    "renormalize",
+    "retriable",
+    "sauron's",
```

Comment:
As much as I love the Hobbit and Lord of the Rings, I really have to question "sauron's" being in public docs.

Reply:
...though, IIRC, he was never called by name in The Hobbit - only referred to as the Necromancer.

Reply:
Hah, you've added it :)

Reply:
Heh. I see - it's a string literal in a sample. It works in context, though I wonder if it should just use `//cspell:` prefixes in that one case rather than a blanket exception.

Reply:
The tricky part is that we would need them both in C# and md file.

Reply:
Done.

Reply:
Wonder if we could ignore code fragment regions since they'd have already been spell-checked. /cc @danieljurek

Reply:
We can try and write a regex that would match end exclude snippets from md files but I would try and see how many occurrences of this we would have.